### PR TITLE
adding logic comments

### PR DESCRIPTION
**Pull request by lanefoxwood**
These comments are pseudo-code before actually finishing the project. they map out my logic process and allow me to separate the syntax and logic into more easily manageable tasks.

### CODE REVIEW

File: Mutations.js [hunk 1]
```diff
@@ -20,4 +20,57 @@ Lastly, ["Alien", "line"], should return true because all of the letters in line
 3. there is a edge case here, where either arr[0] is reversed, or arr[1] is. need an || to reverse one and .test() again.
 4. OK, found another edge case...["Mary", "Army"] should return true, so if all chars of  arr[1] are anywhere within arr[0], return true.
 4.5 I'm thinking my Regex should look for the char set even if they aren't right next to eachother, could be my test im doing with .test()
-*/
\ No newline at end of file
+*/
+
+// adding comments to demonstrate logic process
+
+// this program is going to accept an array as an argument, which contains two strings as the two elements. It will return a boolean. It returns true if the first string contains ALL of the letters of the second string, regardless of case.
```
Comment (lanefoxwood) [inline Mutations.js:27]: looks like things start off well here

File: Mutations.js [hunk 2]
```diff
@@ -20,4 +20,57 @@ Lastly, ["Alien", "line"], should return true because all of the letters in line
...
+};
+
+// an even more concise way of doing the above would be the following:
+/* 
+  return array[0].toLowerCase().includes(array[1].toLowerCase())
+*/
+
+// the only problem would be the edge cases? because if contains all the letters, but the letters are in a mixed up order, it actually should return true. However, the way I;ve written the algorightm using the .includes() method does not account for this. 
+
+//======================================================================================
```
Comment (lanefoxwood) [inline Mutations.js:50]: seems a bit excessive at this point. an exercise, though.

File: longestWord.js [hunk 3]
```diff
@@ -19,4 +19,34 @@ Your response should be a number.*/
...
+
+// this function accepts a string and gives you the length of the longest word
+
+// TODO i can use the string.length() method to find the length of every word within the string
+
+//TODO: i can use the Math.max() method to give the highest length, AKA the longest word.
+
+//TODO: define this function:
+
+function longestWord(string){
```
Comment (lanefoxwood) [inline longestWord.js:34]: ok, good

File: repeatString.js [hunk 4]
```diff
@@ -20,3 +20,38 @@ Repeat a given string str (first argument) for num times (second argument). Retu
 1. take a blank string and concat the str onto it. CHECK
 2. update the value of the blank string to reflect the concat. +=...? CHECK
 3. loop over that until done...? */
+
+
+// adding comments to demonstrate logic process
+
+// this function should receive a string as a first argument and a number as a second argument. it should repeat the string the number of times given. if the number is 0 or less than zero, return an empty string....BIG HINT!!!
+
+//BONUS: make it case-sensitive?
```
Comment (lanefoxwood) [inline repeatString.js:29]: how would you implement this?

**Review by lanefoxwood — COMMENTED** [review]
overall, pretty solid.
